python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -76,17 +76,21 @@ def to_file(j, fobj=None, ignore_subs=''):
 # * AssertionError (corruption: multiple objects in <root>...</root>
 # * Exception (probably corrupted data problem)
 
-def _raw_from_file(f):
+def _raw_from_file(f, obj):
     # logger.debug('----------------------------')
     ###logger.debug('Parsing file: %s',f.name)
     xml_content = f.read()
-    obj, errors = Loader().parse(xml_content)
+    xml_lines = xml_content.splitlines()
+    for line in xml_lines:
+        line = line.lstrip()
+    xml_content = '\n'.join(xml_lines)
+    obj, errors = Loader(obj).parse(xml_content)
```

Comment:
Passing the object allows for the XML to be read from disk directly into the Job object in question, (or other GangaObject)
The string parsing was to remove whitespace to see how it effected file-size and loading speed.
Answer is not very much unfortunately, but I suppose every little helps.